tinuous

```diff
@@ -217,6 +217,15 @@ class GHABuildLog(BuildLog):
     def download(self, path_template: str) -> None:
         p = Path(path_template.format_map(self.path_fields()))
         p.mkdir(parents=True, exist_ok=True)
+        if any(p.iterdir()):
+            log.info(
+                "Logs for %s (%s) #%s already downloaded to %s; skipping",
+                self.workflow_file,
+                self.workflow_name,
+                self.number,
+                p,
+            )
+            return
```

Comment:
what if prior run downloaded some but not all (crashed, CI run was still ongoing) logs?

Reply:
I assume that the script is always going to be run by something (either Datalad or GitHub Actions) that discards newly-downloaded logs when an error/crash occurs.  `tinuous` already skips any CI runs that are still in progress.

Reply:
since skips "in progress" that is great.  For the other, filed dedicated https://github.com/con/tinuous/issues/12 and lets proceed